src/request_queue.js

```diff
@@ -193,17 +193,24 @@ const getRequestId = (uniqueKey) => {
  * @hideconstructor
  */
 export class RequestQueue {
-    constructor(queueId, queueName) {
+    constructor(queueId, queueName, clientId = cryptoRandomObjectId()) {
```

Comment:
clientKey

Reply:
BTW if user passes `clientId=""`, the default will not be used and the call will fail later.